policyengine_uk/variables/gov/dfe/extended_childcare_entitlement/conditions/extended_childcare_entitlement_work_condition.py

```diff
@@ -0,0 +1,59 @@
+from policyengine_uk.model_api import *
+
+
+class extended_childcare_entitlement_work_condition(Variable):
+    value_type = bool
+    entity = Person
+    label = "Work conditions for extended childcare entitlement"
+    documentation = "Whether the person/couple meets work requirements for extended childcare entitlement"
+    definition_period = YEAR
+
```

Comment:
Can simplify this by limiting to parents. Also then don't need the is_child pieces.
```suggestion
    defined_for = "is_parent"

```